modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -180,7 +180,10 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
      * @param ins vector of inputs to process.
      * @sa gin
      */
-    GAPI_WRAP void setSource(GRunArgs &&ins);
+    void setSource(GRunArgs &&ins);
+
+    /// @private -- Exclude this function from OpenCV documentation
+    GAPI_WRAP void setSource(const cv::detail::ExtractArgsCallback& callback);
```

Comment:
This is tested in `python`. Does it make sense to add `c++` test ?

Reply:
+

Reply:
Added